ops/terraform/env/mgmt/variables.tf

```diff
@@ -0,0 +1,5 @@
+variable "enable_cloudtamer_deployment" {
+  type        = bool
+  description = "Allows the creation of cloudtamer specific resources, specifically, the attachment of IAM Paths and permission boundaries"
+  default     = false
+}
```

Comment:
Honestly, this can probably just go away. See some other comments I've made for the full _why_, but basically we'll avoid attaching the PB in this ticket and do that work later on.